stubs/oauthlib/oauthlib/oauth1/__init__.pyi

```diff
@@ -0,0 +1,31 @@
+from .rfc5849 import (
+    SIGNATURE_HMAC as SIGNATURE_HMAC,
+    SIGNATURE_HMAC_SHA1 as SIGNATURE_HMAC_SHA1,
+    SIGNATURE_HMAC_SHA256 as SIGNATURE_HMAC_SHA256,
+    SIGNATURE_HMAC_SHA512 as SIGNATURE_HMAC_SHA512,
+    SIGNATURE_PLAINTEXT as SIGNATURE_PLAINTEXT,
+    SIGNATURE_RSA as SIGNATURE_RSA,
+    SIGNATURE_RSA_SHA1 as SIGNATURE_RSA_SHA1,
+    SIGNATURE_RSA_SHA256 as SIGNATURE_RSA_SHA256,
+    SIGNATURE_RSA_SHA512 as SIGNATURE_RSA_SHA512,
+    SIGNATURE_TYPE_AUTH_HEADER as SIGNATURE_TYPE_AUTH_HEADER,
+    SIGNATURE_TYPE_BODY as SIGNATURE_TYPE_BODY,
+    SIGNATURE_TYPE_QUERY as SIGNATURE_TYPE_QUERY,
+    Client as Client,
+)
+from .rfc5849.endpoints import (
+    AccessTokenEndpoint as AccessTokenEndpoint,
+    AuthorizationEndpoint as AuthorizationEndpoint,
+    RequestTokenEndpoint as RequestTokenEndpoint,
+    ResourceEndpoint as ResourceEndpoint,
+    SignatureOnlyEndpoint as SignatureOnlyEndpoint,
+    WebApplicationServer as WebApplicationServer,
+)
+from .rfc5849.errors import (
+    InsecureTransportError as InsecureTransportError,
+    InvalidClientError as InvalidClientError,
+    InvalidRequestError as InvalidRequestError,
+    InvalidSignatureMethodError as InvalidSignatureMethodError,
+    OAuth1Error as OAuth1Error,
+)
+from .rfc5849.request_validator import RequestValidator as RequestValidator
```

Comment:
Should __init__ unused imports be deleted as well? Wasn't sure if they're somehow special

Reply:
In `__init__.py` they usually make sense. If there is an `__all__` field, this is a good indication of what should be exported.

Reply:
There isn't an `__all__` field, unfortunately. Should I leave this as-is?

Reply:
Sounds good.